src/App.tsx

```diff
@@ -1,25 +1,151 @@
 import React from 'react';
+import { useState } from 'react';
 import './App.css';
-import { getNumbers } from './utils';
+import { getNumbers, getPages, getArrayItems, getItemsForPrint } from './utils';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
+type PageChangeEvent =
+  | React.ChangeEvent<HTMLSelectElement>
+  | React.MouseEvent<HTMLAnchorElement>;
+// console.log(itemsForPrint);
+
+interface PaginationRrops {
+  total: string[];
```

Comment:
The `total` prop should be a number representing the total number of items, not a `string[]`.